freecell/Makefile

```diff
@@ -9,7 +9,7 @@ ADDONS	=
 
 CC      = gcc
 
-CFLAGS	= -O6 -Wall -g -ansi $(TYPE) $(ADDONS)
+CFLAGS	= -O6 -Wall -g $(TYPE) $(ADDONS)
```

Comment:
Hehe, the original author probably didn't know that `-O3` is the maximum optimization level. If you like you can change it since you're touching these lines anyway, but feel free to ignore.

Reply:
oh, good catch and this is funny, it's the first time I saw something like this.